test/util/test_case.hpp

```diff
@@ -28,15 +28,22 @@ namespace ngraph
 {
     namespace test
     {
+        /// \brief Indicates which version of backend (dynamic or static) should be using in
+        /// NgraphTestCase
+        enum class BackendMode
+        {
+            // Use static version of backend
+            STATIC,
+            // Use dynamic version of backend
+            DYNAMIC
+        };
+
         class NgraphTestCase
         {
         public:
             NgraphTestCase(const std::shared_ptr<Function>& function,
-                           const std::string& backend_name)
-                : m_function(function)
-                , m_backend(ngraph::runtime::Backend::create(backend_name))
-            {
-            }
+                           const std::string& backend_name,
+                           BackendMode mode = BackendMode::STATIC);
```

Comment:
I would consider changing the default to `DYNAMIC`

Reply:
@postrational For me is good attitude, all "static backend" based tests pass

Reply:
Changing default mode from `STATIC` to `DYNAMIC` will break alignment with backend creation. Such behavior will confuse an user.

Reply:
After internal discussion `STATIC` version of backend is default at this stage of dynamic support development.